File: src/main/java/xyz/oribuin/lilori/commands/music/CmdVolume.java
package xyz.oribuin.lilori.commands.music;

import xyz.oribuin.lilori.managers.command.Command;
import xyz.oribuin.lilori.managers.command.CommandEvent;
import xyz.oribuin.lilori.managers.music.GuildMusicManager;
import xyz.oribuin.lilori.managers.music.TrackManager;
import xyz.oribuin.lilori.managers.music.TrackScheduler;

import java.util.Collections;

public class CmdVolume extends Command {

    public CmdVolume() {
        this.name = "Volume";
        this.aliases = Collections.emptyList();
        this.description = "Change the volume of the volume";
    }

    @Override
    public void executeCommand(CommandEvent event) {
        TrackManager tm = new TrackManager(event.getGuild());
        String[] args = event.getMessage().getContentRaw().split(" ");

        if (event.getMember().getVoiceState() == null || !event.getMember().getVoiceState().inVoiceChannel()) {
            event.reply(event.getAuthor().getAsMention() + ", Could not change volume since you are not in the voice channel");
            return;
        }

        try {
            int volume = Integer.parseInt(args[1]);

            tm.getMusicManager().player.setVolume(volume);
            event.reply(event.getAuthor().getAsMention() + "Successfully changed volume to " + volume);
        } catch (NumberFormatException ex) {
            event.reply(event.getAuthor().getAsMention() + ", Please include the correct arguments." + event.getPrefix() + this.getName() + " <volume>");
        }
    }
}

File: src/main/kotlin/xyz/oribuin/lilori/commands/global/music/CmdVolume.kt
package xyz.oribuin.lilori.commands.global.music

import xyz.oribuin.lilori.handler.Command
import xyz.oribuin.lilori.handler.CommandEvent
import xyz.oribuin.lilori.managers.music.TrackManager.Companion.getInstance

class CmdVolume : Command() {
    init {
        name = "Volume"
        aliases = emptyList()
        description = "Change the volume of the volume"
    }

    override fun executeCommand(event: CommandEvent?) {
        (event?: return)
        val tm = getInstance(event.guild)
        (tm?: return)

        val args = event.message.contentRaw.split(" ").toTypedArray()
        if (args.size < 2) {
            event.reply("**Current Volume: ${tm.musicManager.player.volume}**")
            return
        }
        if (event.member!!.voiceState == null || !event.member!!.voiceState?.inVoiceChannel()!!) {
            event.reply("${event.author.asMention}, Could not change volume since you are not in the voice channel")
            return
        }

        try {
            val volume = args[1].toInt()
            tm.musicManager.player.volume = volume

            event.reply(event.author.asMention + ", Successfully changed volume to " + volume)
        } catch (ex: NumberFormatException) {
            event.reply(event.author.asMention + ", Please include the correct arguments." + event.prefix + name + " <volume>")
        }
    }
}